multiqc/modules/seqyclean/seqyclean.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re
+
+from multiqc import config
+from multiqc.plots import linegraph, bargraph, table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+import traceback
+
+class MultiqcModule(BaseMultiqcModule):
+	def __init__(self):
+		# Initialise the parent object
+		super(MultiqcModule, self).__init__(name='Seqyclean', anchor='seqyclean',
+		href="http://www.awesome_bioinfo.com/my_module",
+		info="These graphs analyze the seqyclean files created during the pipeline. There are two associated graphs. The first one shows the total number of pairs kept and discarded. The second breaks down the reasons why certain pairs were discarded.")
+
+		# Parse logs
+		self.seqyclean = dict()
+		self.data_total = {}
+		self.data_breakdown = {}
```

Comment:
I don't think that you need two different variables here. The plotting functions ignore any keys which are not specified, so usually you can just keep everything in one `dict` and just plot the keys of interest.